08.ls_object/format.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+class Format
+  def initialize(sorted_current_dir_files)
+    @sorted_current_dir_files = sorted_current_dir_files
+  end
+
+  def shortformat
+    sliced_sorted_current_dir_files = slice_dir(@sorted_current_dir_files)
+
+    transposed_sliced_sorted_current_dir_files = sliced_sorted_current_dir_files.transpose
+
+    transposed_sliced_sorted_current_dir_files.each do |array|
+      blanc_number = @sorted_current_dir_files.max_by(&:length).length + 10
+
+      print array[0] + ' ' * (blanc_number - array[0].length)
+      print array[1] + ' ' * (blanc_number - array[1].length)
+      print array[2]
+      puts "\n"
+    end
+  end
+
+  def longformat
+    total = 0
+    file_data = @sorted_current_dir_files.map do |file|
+      file_details = File.stat(file)
+      file_mode = file_details.mode.to_s(8)
+
+      filetype_and_permission = convert_to_filetype_and_permission(file_mode)
+
+      nlink = file_details.nlink
+
+      username = Etc.getpwuid(file_details.uid).name
+      groupname = Etc.getgrgid(file_details.gid).name
+
+      bytesize = file_details.size
+
+      timestamp = file_details.mtime.strftime('%-m %e %H:%M')
+
+      total += file_details.blocks
```

Comment:
空行の入る基準がちょっと気になるというか、この程度の処理であれば空行なしのほうが見やすそうな気がしました。
```suggestion
      file_details = File.stat(file)
      file_mode = file_details.mode.to_s(8)
      filetype_and_permission = convert_to_filetype_and_permission(file_mode)
      nlink = file_details.nlink
      username = Etc.getpwuid(file_details.uid).name
      groupname = Etc.getgrgid(file_details.gid).name
      bytesize = file_details.size
      timestamp = file_details.mtime.strftime('%-m %e %H:%M')
      total += file_details.blocks
```